src/components/Filters/FilterRow/FilterRow.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { FlexContainer } from '../../FlexContainer';
+import { StateButton } from '../StateButton';
+import { Select } from '../Select';
+import { Input } from '../Input';
+import { WhereOption } from '../WhereOption';
+import { FilterRowProps, SplitFieldProps } from './FilterRow.types';
+import { OptionPropType } from '../Select/Select.types';
+import { pxToRem } from '../../../utils/helpers';
+import operatorOptions from '../data/operator-options.json';
+
+const Container = styled(FlexContainer)`
+  margin-bottom: ${pxToRem(8)};
+  &:last-of-type {
+    margin-bottom: ${pxToRem(0)};
+  }
+`;
+
+const SplitField = styled.div`
```

Comment:
if you do 
```js
const SplitField = styled.div<SplitFieldProps>
```
you don't have to pass type annotation in the `width` property

Reply:
👍 Good to keep types syntax consistent